store/displayPreferences.ts

```diff
@@ -1,116 +1,108 @@
-import Vue from 'vue';
 import { DisplayPreferencesDto } from '@jellyfin/client-axios';
-import { MutationTree, ActionTree, GetterTree } from 'vuex';
+import { MutationTree, ActionTree } from 'vuex';
+import { merge, toString as toStr } from 'lodash';
+import { boolean } from 'boolean';
```

Comment:
Why do we need to import this?

Reply:
That method was our custom one, MrTim decided a custom library was a good idea because it accepts a lot of possible value and handles already lowercasing and such for us:

https://www.npmjs.com/package/boolean

Imo I think it's a good addition, it has an small footprint and keeps code simple.